website/content/docs/api-gateway/index.mdx

```diff
@@ -5,9 +5,9 @@ description: >-
   Consul API Gateway enables external network client access to a service mesh on Kubernetes and forwards requests based on path or header information. Learn about how the k8s Gateway API specification configures Consul API Gateway so you can control access and simplify traffic management.
 ---
 
-# API Gateway for Kubernetes Overview
+# API Gateway on Kubernetes overview
```

Comment:
We consistently use "for Kubernetes" across the set of docs in `website/content/docs/api-gateway`. I would consistently use "on" or "for" everywhere. If using "on", I'd update everything touched in https://github.com/hashicorp/consul/pull/16360

Reply:
```suggestion
# API Gateway for Kubernetes overview
```